Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 name = "indexer-explorer"
-version = "0.10.14"
+version = "0.10.15"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2021"
 
 [dependencies]
-actix = "=0.11.0-beta.2"
+actix = "=0.13.0"
 actix-rt = "=2.2.0"  # remove it once actix is upgraded to 0.11+
```

Comment:
According to the comment, we should be able to remove this line (otherwise, update the comment)